cmd/bootstrap/async_replication.go

```diff
@@ -112,10 +112,13 @@ func bootstrapAsyncReplication(ctx context.Context) error {
 		return nil
 	}
 
+	requireClone := getCloneRequired()
```

Comment:
`getCloneRequired()` vs `isCloneRequired()` … these two functions are very similar in naming and its not obvious what they do differently.